src/bin/import.js

```diff
@@ -8,23 +8,22 @@
 
 var webgme = require('../../webgme'),
     program = require('commander'),
-    BRANCH_REGEXP = new RegExp('^[0-9a-zA-Z_]*$'),
     FS = require('fs'),
     path = require('path'),
     openContext,
     Serialization,
-    jsonProject;
+    jsonProject,
+    gmeConfig = require(path.join(process.cwd(), 'config')),
+    logger = webgme.Logger.create('gme:bin:import', gmeConfig.bin.log),
+    REGEXP = webgme.REGEXP,
+    openContext = webgme.openContext,
+    Serialization = webgme.serializer;
 
 
-openContext = webgme.openContext;
-Serialization = webgme.serializer;
-
-
-var importProject = function (Storage, gmeConfig, projectId, jsonProject, branchName, overwrite, callback) {
+var importProject = function (Storage, _gmeConfig, projectId, jsonProject, branchName, overwrite, callback) {
```

Comment:
To distinguish between the global gmeConfig (used when the script is running as bin) and the configuration passed via importProject (e.g. in tests).